modules/distr-tracing-otel-migrating-from-jaeger-with-sidecars.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * distr-tracing-otel-migrating.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-migrating-from-jaeger_{context}"]
+= (Temporary title) Migrating from {JaegerName} to {OTELName} with sidecars
+
+.Prerequisites
+
+* Existing instrumentation of applications uses {JaegerName} for tracing.
+* {OTELName} is installed.
+
+.Procedure
+
+The Red Hat Red Hat OpenShift Distributed Tracing Data Collection Operator can inject sidecars into Deployment workloads.
```

Comment:
@iblancasa, since we have `:OTELName: Red Hat OpenShift distributed tracing data collection`, do you agree with the following suggestion?
```suggestion
The {OTELName} Operator can inject sidecars into Deployment workloads.
```